lib/engine/step/train.rb

```diff
@@ -90,7 +90,6 @@ def process_sell_shares(action)
         @last_share_sold_price = action.bundle.price_per_share
         super
         @corporations_sold << action.bundle.corporation
-        @round.recalculate_order
```

Comment:
This needs to be left

Reply:
Doesn't `super` call the one in `emergency_money.rb`?